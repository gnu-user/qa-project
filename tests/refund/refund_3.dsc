TITLE
    Refund Buyer Input (Valid Input, buyer exists)
DESCRIPTION
    Run the login transaction using an admin account. Initiate the refund 
    transaction. When prompted, enter a buyer which exists. Verify that the 
    buyer is accepted without error.
DEPENDENCIES
    refund_1
