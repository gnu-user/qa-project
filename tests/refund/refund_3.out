Welcome.
> 
Enter username: 
Username accepted.
Available tickets file read successfully.
>
Enter the buyer's username: 
Enter the seller's username: 